formats.sh

```diff
@@ -4,21 +4,25 @@
 # without updating codebase.
 
 
-res_pip_list=$(pip freeze)
 missing_dependencies=()
-if [ ! "$(echo $res_pip_list | grep black)" ] ; then
+command -v black &> /dev/null
+if [ $? -eq 1 ] ; then
   missing_dependencies+=(black)
 fi
-if [ ! "$(echo $res_pip_list | grep blackdoc)" ] ; then
+command -v blackdoc &> /dev/null;
```

Comment:
As hvy said, we don't need a semicolon here.

Reply:
Oops, Indeed. Thank you for your comments on the lines.